docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md

```diff
@@ -0,0 +1,211 @@
+---
+author:
+  name: Andrew Leap
+  email: andyleap@gmail.com
+description: 'Alpine Linux is a small, security oriented linux distro.  This guide details installing and configuring Alpine Linux on a KVM Linode'
```

Comment:
Should this be "Linux"? Also, with KVM Linodes being the new default, does it make sense to just say "on a Linode", and maybe add a note to the top about differences with Xen for legacy servers?